src/ResXManager.View/Tools/ResXFileWatcher.cs

```diff
@@ -65,6 +65,9 @@ protected override void OnFilesChanged()
                 if (language.HasChanges)
                     continue;
 
+                if(_resourceManager.Configuration.SortFileContentOnSave)

```

Comment:
- Please stick to projects formatting rules (white-space, braces)
- code dup: dereferencing config three times